tests/faces/test_cylindrical_face3d.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import volmdlr
+from volmdlr import faces, edges
+
+
+class TestCylindricalFace3D(unittest.TestCase):
```

Comment:
You developed `linesegment_intersections` for `CylindricalSurface3D` and not `CylindricalFace3D` !